index.js

```diff
@@ -5,7 +5,7 @@ const {
   collectCalls,
   collectCallsAndPuts,
   createSagaTestEngine,
-} = require('./src')
+} = require('./build')
 
 module.exports = {
   default: createSagaTestEngine,
```

Comment:
If we want to make the entire library ES5 compatible, we should probably not rely on `const` or object destructuring in `index.js`, right? (Not entirely sure how it works TBH :-) )

```js
var SagaTestEngine = require('./build')

var collectPuts = SagaTestEngine.collectPuts
var collectCalls = SagaTestEngine.collectCalls
var collectCallsAndPuts = SagaTestEngine.collectCallsAndPuts
var createSagaTestEngine = SagaTestEngine.createSagaTestEngine

module.exports = {
  default: createSagaTestEngine,
  createSagaTestEngine: createSagaTestEngine,
  collectPuts: collectPuts,
  collectCalls: collectCalls,
  collectCallsAndPuts: collectCallsAndPuts,
}
```